src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoFirstDayOfWeek.cs

```diff
@@ -12,8 +12,8 @@ public class DateTimeFormatInfoFirstDayOfWeek
         public static IEnumerable<object[]> FirstDayOfWeek_TestData()
         {
             yield return new object[] { DateTimeFormatInfo.InvariantInfo, DayOfWeek.Sunday };
-            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, DayOfWeek.Sunday };
-            yield return new object[] { new CultureInfo("fr-FR").DateTimeFormat, DayOfWeek.Monday };
+            yield return new object[] { new CultureInfo("en-US", false).DateTimeFormat, DayOfWeek.Sunday };
```

Comment:
This change is fine. 

I assume you have tested it with changing the machine settings to Monday as the first day of the week while you have en-US as default culture

Reply:
honestly, no. let me do that ;)

Reply:
yes, the change works as expected. thanks @tarekgh for your help.

Reply:
Thanks @ViktorHofer for your help fixing the test